src/cr/cube/matrix.py

```diff
@@ -957,6 +999,18 @@ def _addend_vectors(self):
             if i in self._subtotal.addend_idxs
         )
 
+    @lazyproperty
```

Comment:
These also seem as if they belong to the `_InsertionVector` and not row or column